apps/frontend/src/libs/components/input/input.tsx

```diff
@@ -14,10 +14,11 @@ import styles from "./styles.module.css";
 type Properties<T extends FieldValues> = {
 	control: Control<T, null>;
 	errors?: FieldErrors<T>;
+	hasVisuallyHiddenLabel?: boolean;
 	iconName?: IconName;
 	label: string;
 	name: FieldPath<T>;
-	onIconClick?: () => void;
+	onIconClick?: React.MouseEventHandler<HTMLButtonElement>;
```

Comment:
Do we really need this change?

Reply:
Changing props in Button causes a linter error here.